### DISCUSSION TITLE
Can we have bionic reading feature in Thorium?

### DISCUSSION THREAD

**Discussion by mohitmujawdiya** [edrlab/thorium-reader#1739] [Ideas]
Hi, is an implementation like bionic reading possible on Thorium? Here is an example:

![image](https://user-images.githubusercontent.com/61835205/169841610-c58ec79f-a528-40ca-ba74-99ca677215a0.png)

This method helps people read efficiently. An extension has already been made for Chrome. Here is the repo in case the code is helpful. (https://github.com/ansh/bionic-reading)

Bionic Reading is patented in France and trademarked in some other countries, but I am sure it won't be much of an issue (https://www.reddit.com/r/ObsidianMD/comments/uv995v/bionic_reading_seems_great_but_patent_is_a_bummer/)

**Comment by danielweck** [comment]
> Bionic Reading is patented in France and trademarked in some other countries, but I am sure it won't be much of an issue

I am no lawyer, but I wouldn't be so sure about that 😄 

Note that Thorium ships with a typeface / web font that has been shown to facilitate reading in people with dyslexia ( https://opendyslexic.org ), but this is not a universally-acclaimed solution, in the sense that not every reader benefit from this kind of special glyph rendering.

In my humble opinion, we should hold similar reservations about this "bionic" text style. In fact I have already seen some criticism of it, where it makes reading harder, not easier (the human brain doesn't use just word-by-word pattern recognition when our vision scans a flow of textual information)

There is prior art / research on this subject. Just to name one: https://www.microsoft.com/en-gb/education/products/learning-tools
(try the "Immersive Reader" demo with highlighted "parts of speech" and syllables)

So, personally I do not expect the Thorium team to place a high development priority on this feature request.

**Comment by llemeurfr** [comment]
Triage: convert to discussion.
